internal/txvalidation/config_tx_validator.go

```diff
@@ -27,7 +27,7 @@ type configTxValidator struct {
 	logger          *logger.SugarLogger
 }
 
-func (v *configTxValidator) validate(txEnv *types.ConfigTxEnvelope) (*types.ValidationInfo, error) {
+func (v *configTxValidator) Validate(txEnv *types.ConfigTxEnvelope) (*types.ValidationInfo, error) {
```

Comment:
Why only in `configTxValidator` method `Validate` defined as public?

Reply:
Right now we don't have immediate plans to do the validation of other type of TXs pre-order, only config-TX. The validation of data TXs pre-order for example, may occur in the tx-reorderer component for example. I think that when we get to it, we'll have to review the requirements again, and maybe require some implementation changes.  Therefore I did not  want to make these methods public at this point.

The `configTxValidator.Validate`, on the other hand, will be used in the next PR that I plan.